apps/web/pages/auth/login.tsx

```diff
@@ -182,8 +182,6 @@ export default function Login({
                     StartIcon={FaGoogle}
                     onClick={async (e) => {
                       e.preventDefault();
-                      // track Google logins. Without personal data/payload
-                      telemetry.event(telemetryEventTypes.googleLogin, collectPageParameters());
```

Comment:
@G3root for your info, i'm using this opportunity to remove SAML and Google Sign in tracking. It doesn't provide any useful insights and will lower our tracked events per month